float64 latitude
float64 longitude
float64 radius
float64 height